hystrix-core/src/main/java/com/netflix/hystrix/strategy/HystrixPlugins.java

```diff
@@ -295,6 +345,32 @@ private static Object getPluginImplementationViaArchaius(Class<?> pluginClass) {
             return null;
         }
     }
+    
+    
+
+    private static HystrixDynamicProperties resolveDynamicProperties(ClassLoader classLoader) {
```

Comment:
IMO, I think it's valuable to add a logger.info call here on each possible return.  This will aid debugging as users can understand where the `HystrixDynamicProperties` instance is getting loaded from.

Something like : 
`logger.info("Created HystrixDynamicProperties instance from System property named \"hystrix.plugin.HystrixDynamicProperties.implementation\");

or

logger.info("Created HystrixDynamicProperties instance by loading from ServiceLoader.  Using class : " + hp.getClass().getSimpleName()");

As examples of verbosity.  

What do you think?